lib/runners/bf.js

```diff
@@ -0,0 +1,34 @@
+var shovel = require('../shovel'),
+    util = require('../util');
+
+module.exports.run = function run(opts, cb) {
+  shovel.start(opts, cb, {
+    solutionOnly: function(runCode) {
+      var solutionFile = util.writeFileSync('/home/codewarrior', 'code.b', opts.solution, true);
+      runCode({
+        name: 'bf',
+        args: [solutionFile]
+      });
+    },
+    testIntegration: function(runCode, fail) {
+      switch (opts.testFramework) {
+        case 'cw-2':
+          var fixture = `
+require(\'/runner/frameworks/javascript/cw-2\');
+const runBF = require(\'/runner/frameworks/bf/run-bf.js\');
+${opts.fixture};
```

Comment:
Oops, sorry, I forgot to update those, my bad 😛 Those unnecessary backslashes should be removed now :smile: